src/actions/songs/recommend-songs.js

```diff
@@ -0,0 +1,87 @@
+const song = require("../../models").song,
+	genre = require("../../models").genre,
+	user = require("../../models").user,
+	listened = require("../../models").previously_listened,
+	songResult = require("../../models").likert_template_result,
+	maxSongs = 10;
+
+var songs, temp = [], amountResults
+
+module.exports = (genreId, moodId) => {
+	listenedMusic(genreId, moodId)
+	newMusic(genreId, moodId)
+	return songs;
```

Comment:
The `songs` is coming from an outer scope, which is questionable on itself. Adding to that, it is not being modified after its original `[]`, thus this will always return `[]`;